tools/editor/scenes_dock.cpp

```diff
@@ -1416,6 +1428,10 @@ void ScenesDock::_files_list_rmb_select(int p_item,const Vector2& p_pos) {
 
 	file_options->add_item(TTR("Delete"),FILE_REMOVE);
 	//file_options->add_item(TTR("Info"),FILE_INFO);
+
+	file_options->add_separator();
+	file_options->add_item(TTR("Show In System"),FILE_SHOW_IN_EXPLORER);
```

Comment:
I think the common function is
Globals::get_singleton()->globalize_path(res_path)

On Sat, May 21, 2016 at 8:25 AM, Ignacio Etcheverry <
notifications@github.com> wrote:

> In tools/editor/scenes_dock.cpp
> <https://github.com/godotengine/godot/pull/4700#discussion_r64133113>:
>
> > @@ -1416,6 +1428,10 @@ void ScenesDock::_files_list_rmb_select(int p_item,const Vector2& p_pos) {
> >
> >  	file_options->add_item(TTR("Delete"),FILE_REMOVE);
> >  	//file_options->add_item(TTR("Info"),FILE_INFO);
> > +
> > +	file_options->add_separator();
> > +	file_options->add_item(TTR("Show In System"),FILE_SHOW_IN_EXPLORER);
>
> IMO the name may be a bit confusing. Something like *"Show In File
> Manager"* could be better.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/godotengine/godot/pull/4700/files/5ad648097930e46df3d23f514e8bc08fa00b31f5#r64133113>
>